library/src/main/scala/scala/scalajs/js/Dynamic.scala

```diff
@@ -76,7 +76,27 @@ sealed trait Dynamic extends js.Any with scala.Dynamic {
 /** Factory for dynamically typed JavaScript values. */
 object Dynamic {
   /** Dynamic view of the global scope. */
-  def global: js.Dynamic = throw new java.lang.Error("stub")
+  @js.native
+  @JSGlobalScope
+  object global extends js.Any with scala.Dynamic { // scalastyle:ignore
+    /** Calls a top-level method (in the global scope). */
+    @JSBracketCall
+    def applyDynamic(name: String)(args: js.Any*): js.Dynamic = js.native
+
+    /** Reads a top-level variable (in the global scope). */
+    @JSBracketAccess
+    def selectDynamic(name: String): js.Dynamic = js.native
+
+    /** Writes to a top-level variable (in the global scope). */
+    @JSBracketAccess
+    def updateDynamic(name: String)(value: js.Any): Unit = js.native
+
+    /** Cannot be called--provides a compile-time error instead of a silent
+     *  run-time error if one tries to do `js.Dynamic.global(something)`.
+     */
+    @deprecated("The global scope cannot be called as function.", "forever")
+    def apply(args: js.Any*): js.Dynamic = js.native
```

Comment:
Because otherwise `js.Dynamic.global(foo)` silently desugars to `js.Dynamic.global.applyDynamic("apply")(foo)`, which compiles and links, but doesn't do what it looks like it's going to do.

Wasn't the comment explicit enough?